src/Plutarch/Extra/Function.hs

```diff
@@ -64,3 +67,49 @@ pflip ::
         s
         ((a :--> b :--> c) :--> b :--> a :--> c)
 pflip = phoistAcyclic $ plam $ \f y x -> f # x # y
+
+infixr 8 #.*
+infixr 8 #.**
+infixr 8 #.***
```

Comment:
I find it more readable to put fixities after individual definitions, but this is a minor issue.

Reply:
Will fix!